x-pack/test/observability_functional/apps/observability/alerts/add_to_case.ts

```diff
@@ -40,15 +40,15 @@ export default ({ getService, getPageObjects }: FtrProviderContext) => {
         await observability.users.restoreDefaultTestUserRole();
       });
 
-      it('renders case options in the overflow menu', async () => {
+      it.skip('renders case options in the overflow menu', async () => {
         await observability.alerts.common.openActionsMenuForRow(0);
         await retry.try(async () => {
           await observability.alerts.addToCase.getAddToExistingCaseSelectorOrFail();
           await observability.alerts.addToCase.getAddToNewCaseSelectorOrFail();
         });
       });
 
-      it('opens a flyout when Add to new case is clicked', async () => {
+      it.skip('opens a flyout when Add to new case is clicked', async () => {
         await observability.alerts.addToCase.addToNewCaseButtonClick();
 
         await retry.try(async () => {
```

Comment:
Are these skipped because they were failing? Should we investigate further before merging?